_includes/getting-started.html

```diff
@@ -288,6 +288,7 @@ <h4 style="font-size: 1.5em;">Using Helm</h4>
             <p class="code-box" style="height: 90px; line-height: 30px;"><code style="position: absolute;">kubectl create ns meshery
               helm repo add meshery https://meshery.io/charts
               helm install meshery meshery/meshery -n meshery
+              helm install meshery-operator meshery/meshery-operator -n meshery 
```

Comment:
@Revolyssup please move this to the line before Meshery's. Recall that I just mentioned that the Operator should be installed first.